src/Tasks/ResolveManifestFiles.cs

```diff
@@ -52,6 +52,8 @@ public sealed class ResolveManifestFiles : TaskExtension
         // if signing manifests is on and not all app files are included, then the project can't be published.
         private bool _canPublish;
         private Dictionary<string, ITaskItem> _runtimePackAssets;
+        // map of satellite assemblies that are included in References
+        private SatelliteRefAssemblyMap satelliteRefAssemblyMap = new SatelliteRefAssemblyMap();
```

Comment:
I found this name unclear later on. Maybe call it `_satelliteAssembliesPassedAsReferences`?

(also nit: our style is that this should start with an `_`)

Reply:
fixed.